tests/unit/amo/pages/TestSearchPage.js

```diff
@@ -153,6 +157,28 @@ describe(__filename, () => {
     sinon.assert.notCalled(fakeDispatch);
   });
 
+  it('dispatches a server redirect when `category` and `addonType` is set', () => {
```

Comment:
why is it "is set" and not "are set"?

Reply:
Because my English is poor (or I wasn't paying attention).